modules/core/src/test/scala/com/azavea/stac4s/CatalogLayerSpec.scala

```diff
@@ -0,0 +1,328 @@
+      val layerUS = getJson("/catalogs/landsat-stac-layers/layers/us/catalog.json").as[StacCatalog].valueOr(throw _)
+      val layerCA = getJson("/catalogs/landsat-stac-layers/layers/ca/catalog.json").as[StacCatalog].valueOr(throw _)
+
+      val item = StacItem(
+        id = "LC81530252014153LGN00",
+        stacVersion = "0.9.0",
+        stacExtensions = List("eo", "view", "layers", "https://example.com/stac/landsat-extension/1.0/schema.json"),
+        geometry = """
+                     |{
+                     |    "type": "Polygon",
+                     |    "coordinates": [
+                     |      [
+                     |        [
+                     |          51.33855,
+                     |          72.27502
+                     |        ],
+                     |        [
+                     |          51.36812,
+                     |          75.70821
+                     |        ],
+                     |        [
+                     |          49.19092,
+                     |          75.67662
+                     |        ],
+                     |        [
+                     |          49.16354,
+                     |          72.39640
+                     |        ],
+                     |        [
+                     |          51.33855,
+                     |          72.27502
+                     |        ]
+                     |      ]
+                     |    ]
+                     |  }""".stripMargin.parseGeoJson[Polygon],
+        bbox = TwoDimBbox(49.16354, 72.27502, 51.36812, 75.67662),
+        properties = Map(
+          "collection"                           -> "landsat-8-l1".asJson,
+          "datetime"                             -> "2014-06-02T09:22:02Z".asJson,
+          "eo:gsd"                               -> 15.asJson,
+          "eo:cloud_cover"                       -> 10.asJson,
+          "view:off_nadir"                       -> 0.asJson,
+          "view:sun_azimuth"                     -> 149.01607154.asJson,
+          "view:sun_elevation"                   -> 59.21424700.asJson,
+          "view:azimuth"                         -> 0.asJson,
+          "landsat:wrs_path"                     -> 153.asJson,
+          "landsat:wrs_row"                      -> 25.asJson,
+          "landsat:earth_sun_distance"           -> 1.0141560.asJson,
+          "landsat:ground_control_points_verify" -> 114.asJson,
+          "landsat:geometric_rmse_model"         -> 7.562.asJson,
+          "landsat:image_quality_tirs"           -> 9.asJson,
+          "landsat:ground_control_points_model"  -> 313.asJson,
+          "landsat:geometric_rmse_model_x"       -> 5.96.asJson,
+          "landsat:geometric_rmse_model_y"       -> 4.654.asJson,
+          "landsat:geometric_rmse_verify"        -> 5.364.asJson,
+          "landsat:image_quality_oli"            -> 9.asJson
+        ).asJsonObject.deepMerge(LayerProperties(List(layerUS.id, layerCA.id)).asJsonObject), // layer extension
```

Comment:
@jisantuc that's how I used it... mb we can think of some nice API later to handle that.